indico/modules/events/models/events.py

```diff
@@ -555,6 +562,17 @@ def short_external_url(self):
         id_ = self.url_shortcut or self.id
         return url_for('events.shorturl', confId=id_, _external=True)
 
+    @property
+    def map_url(self):
+        map_url = self._map_url
```

Comment:
```
if self._map_url:
    return self._map_url:
elif not self.room:
    return None  # or empty string? not sure what's better
return self.room.map_url
```

Reply:
Since `_map_url` defaults to empty string, I think it would be more consistent to do something like this:

```python
def map_url(self):
    if self._map_url:
        return self._map_url
    elif not self.room:
        return ''
    return self.room.map_url or ''
```